src/test/CMakeLists.txt

```diff
@@ -990,3 +990,11 @@ add_ceph_unittest(unittest_weighted_shuffle)
 add_executable(unittest_intarith test_intarith.cc)
 add_ceph_unittest(unittest_intarith)
 #make check ends here
+
```

Comment:
we already have some teuthology tests running for nvmeof. The test that covers HA, is waiting for this PR to be merged, so it can be also promoted. Is that OK?

Reply:
Where are the tests? We generally merge tests with the code in question. This whole arrangement is weird, so we can do something different, but they still need to be presented up front.

Reply:
- We have tests already under https://github.com/ceph/ceph/tree/main/qa/suites/rbd/nvmeof 
- We also have a lot of CI tests in https://github.com/ceph/ceph-nvmeof/, including HA tests. 
- We also have jenkins automation running nightly testing, and on-demand testing for HA and other scenarios.